sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiverOptions.cs

```diff
@@ -40,6 +40,12 @@ public int PrefetchCount
         /// </summary>
         public ServiceBusReceiveMode ReceiveMode { get; set; } = ServiceBusReceiveMode.PeekLock;
 
+        /// <summary>
+        /// A property used to set the <see cref="ServiceBusReceiver"/> ID to identify the client. This can be used to correlate logs
+        /// and exceptions. If <c>null</c> or empty, a random unique value will be used.
+        /// </summary>
```

Comment:
Should we also add a remarks section that describes how this is used in the LinkName?

Reply:
What benefit would that provide customers?  Personally, I see that as leaking implementation details, but I defer to your judgement.

Reply:
Fair point, I withdraw this comment 😆